examples/analog_in/ai_voltage_waveform_finite_acq.py

```diff
@@ -0,0 +1,23 @@
+﻿"""Example of analog input voltage acquisition.
+
+This example demonstrates how to acquire a finite amount
+of data using the DAQ device's internal clock.
+"""
+
+import pprint
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+pp = pprint.PrettyPrinter(indent=4)
+
+
+with nidaqmx.Task() as task:
+    task.ai_channels.add_ai_voltage_chan("Dev1/ai0")
+    task.timing.cfg_samp_clk_timing(1000.0, sample_mode=AcquisitionType.FINITE, samps_per_chan=1000)
+
+    print("Reading Data: ")
+    task.start()
+    data = task.read()
+    pp.pprint(data)
+    task.stop()
```

Comment:
start() and stop() are not required here. task.read() will automatically start/stop the task.